app/src/main/res/values/strings_user_pref_values.xml

```diff
@@ -32,13 +32,15 @@
 
     <string name="pref_default_git_server_port" translatable="false">22</string>
 
+    <string name="pref_default_data_server">https://wacs.bibletranslationtools.org</string>
+
     <string name="pref_default_gogs_api" translatable="false">https://wacs.bibletranslationtools.org/api/v1</string>
 
     <string name="pref_default_media_server" translatable="false">https://api.bibletranslationtools.org</string>
 
     <string name="pref_default_reader_server" translatable="false">https://read.bibleineverylanguage.org</string>
 
-    <string name="pref_default_create_account_url" translatable="false">https://content.bibletranslationtools.org/user/sign_up</string>
+    <string name="pref_default_create_account_url" translatable="false">https://wacs.bibletranslationtools.org/user/sign_up</string>
```

Comment:
Same here.  Please use `content`, not `wacs`.

Reply:
Done.